<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlSystemTracker</class>
 <widget class="QDialog" name="ControlSystemTracker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>omega_d [rad/s]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>zeta (damping)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Overshoot</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberOvershoot">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberDamping">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberW">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Settling Time [s]</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberSettlingTime">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
